rpi-image-builder/aws-iot-fleet-provisioning/provisioning_handler.py

```diff
@@ -0,0 +1,339 @@
+# ------------------------------------------------------------------------------
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0 
```

Comment:
this is MIT-0, while the rest of the source is Apache 2.0…

Reply:
This code comes from a different project That's using MIT-0 license